Project Mars/MarsQA1/MarsQA-1/Steps/Language_Steps.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using MarsQA1.Helper;
+using MarsQA1.Specflow_Pages;
+using TechTalk.SpecFlow;
+
+namespace MarsQA1.Steps;
+
+[Binding]
+public sealed class Language_Steps:Driver
+{ 
+    Language_page languageObj = new Language_page();
```

Comment:
please make use of constructors!

have a look at this https://www.knowledgehut.com/tutorials/csharp/csharp-constructors

Reply:
please fix for all classes